bitbucket-slack-server-integration-plugin/src/main/resources/analytics/whitelist.json

```diff
@@ -40,7 +40,7 @@
 
   "notifications.slack.notification.regular.sent": ["userKeyHash", "teamIdHash", "slackUserIdHash", {
     "notificationKey": ["PRCreated", "PRParticipantUpdated", "PRReviewersUpdated", "PRMerged", "PRDeclined",
-      "PRReopened", "PRDeleted", "PRParticipantApproved", "PRParticipantUnapproved", "PRParticipantReviewed",
+      "PRReopened", "PRDeleted", "PRReScoped", "PRParticipantApproved", "PRParticipantUnapproved", "PRParticipantReviewed",
```

Comment:
This section is related to regular/repository notifications being sent. It isn't planned. Only personal notifications will be sent on `PRReScoped` event. So, this change needs to be reverted.